frontend/src/app/app.component.html

```diff
@@ -1,4 +1,4 @@
-<mat-drawer-container class="container" autosize>
+<!-- <mat-drawer-container class="container" autosize>
   <mat-drawer #drawer class="sidenav">
     <button color="primary" mat-button routerLink="/guide">Guide</button>
```

Comment:
delete commented code.

Reply:
That version is outdated. The most recent one is [here](https://github.com/MiniCRM-Group/MiniCRM/pull/16/files#diff-35d40ad6f8c1f7803af05ca3dd2b6f9a) and has deleted all the commented code.